pxc/jenkins/qa_pxc_cross_version_57_80_test-pipeline.groovy

```diff
@@ -0,0 +1,179 @@
+      parallel {
+        stage('Ubuntu Bionic') {
+          agent {
+            label "min-bionic-x64"
+          }
+          steps {
+            withCredentials([usernamePassword(credentialsId: 'JenkinsAPI', passwordVariable: 'JENKINS_API_PWD', usernameVariable: 'JENKINS_API_USER')]) {
+              run_test()
+            }
+          } //End steps
+        } //End stage Ubuntu Bionic
+        stage('Ubuntu Focal') {
+          agent {
+            label "min-focal-x64"
+          }
+          steps {
+            withCredentials([usernamePassword(credentialsId: 'JenkinsAPI', passwordVariable: 'JENKINS_API_PWD', usernameVariable: 'JENKINS_API_USER')]) {
+              run_test()
+            }
+          } //End steps
+        } //End stage Ubuntu Focal
+        stage('Debian Bullseye') {
+          agent {
+            label "min-bullseye-x64"
+          }
+          steps {
+            withCredentials([usernamePassword(credentialsId: 'JenkinsAPI', passwordVariable: 'JENKINS_API_PWD', usernameVariable: 'JENKINS_API_USER')]) {
+              run_test()
+            }
+          } //End steps
+        } //End stage Debian Bullseye
+        stage('Debian Buster') {
+          agent {
+            label "min-buster-x64"
+          }
+          steps {
+            withCredentials([usernamePassword(credentialsId: 'JenkinsAPI', passwordVariable: 'JENKINS_API_PWD', usernameVariable: 'JENKINS_API_USER')]) {
+              run_test()
+            }
+          } //End steps
+        } //End stage Debian Buster
+        stage('Centos7') {
+          agent {
+            label "min-centos-7-x64"
+          }
+          steps {
+            withCredentials([usernamePassword(credentialsId: 'JenkinsAPI', passwordVariable: 'JENKINS_API_PWD', usernameVariable: 'JENKINS_API_USER')]) {
+              run_test()
+            }
+          } //End steps
+        } //End stage CentOS7
+        stage('Oracle Linux 8') {
+          agent {
+            label "min-ol-8-x64"
+          }
+          steps {
+            withCredentials([usernamePassword(credentialsId: 'JenkinsAPI', passwordVariable: 'JENKINS_API_PWD', usernameVariable: 'JENKINS_API_USER')]) {
+              run_test()
+            }
+          } //End steps
+        } //End stage Oracle Linux 8
+       } //End parallel
+    } //End stage Run tests
+  } //End stages
+} //End pipeline
+
+void run_test() {
+  sh 'echo Downloading PXC LOWER VERSION tarball: \$(date -u)'
+               sh '''
+                    echo "Installing dependencies..."
+                    if [ -f /usr/bin/yum ]; then 
+                    sudo yum -y update
+                    sudo yum install -y git wget tar socat
+                    else
+                    sudo apt install -y git wget socat curl gnupg2 numactl
+                    curl -O https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+                    sudo apt install -y ./percona-release_latest.generic_all.deb
```

Comment:
Why are we not installing percona-release on CentOS based platforms? Is it pre-installed there ?